src/Templates/TodoTemplate/Bit.TodoTemplate/Api/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -99,10 +101,112 @@ public static void AddJwt(this IServiceCollection services, IConfiguration confi
         services.AddAuthorization();
     }
 
+    /// <summary>
+    /// https://docs.microsoft.com/en-us/odata/concepts/queryoptions-overview
+    /// </summary>
+    public class ODataOperationFilter : IOperationFilter
```

Comment:
please move this class to its own file

Reply:
We've a lot of folders in api project and it's crowded already